modules/data.land/R/dataone_download.R

```diff
@@ -24,23 +25,16 @@ dataone_download = function(id, username, CNode = "PROD", lazyLoad = FALSE, quie
   pkg <- dataone::getDataPackage(d1c, id = id, lazyLoad = lazyLoad, quiet = quiet, limit = "1MB") # what is the standard limit for pecan downloads?
   files <- datapack::getValue(pkg, name="sysmeta@formatId")
   n <- length(files) # number of files
-  
-  # fileath to /dbfiles 
-  fp <- "/fs/data1/pecan.data/dbfiles/"
-  
+
   # make new directory within this directory
-  newdir <- paste(fp, "NewData_", username, sep = "")
-  system(paste("mkdir", newdir))
-  
-  # switch to new directory -- unsure if I should do this in R or in unix
-  # system(paste("cd", fp, sep = " ")) 
-  setwd(newdir)
+  newdir <- paste(filepath, "NewData_", username, sep = "")
```

Comment:
Because there's no guarantee that ```filepath``` will end in a / it's better to use file.path than paste

Also, the decision to use the username in the path both breaks with PEcAn conventions and will result in everything a single user ever downloads going into the same directory. Better to use a meaningful identifier, such as the id being passed in. "NewData" also isn't particularly indentifying

How about....
```
newdir <- file.path(filepath, paste0("DataOne_", id))
```